packages/graphql-hooks/src/GraphQLClient.ts

```diff
@@ -45,12 +135,14 @@ class GraphQLClient {
     }
 
     this.cache = config.cache
-    this.headers = config.headers || {}
+    this.headers =
+      config.headers || (typeof Headers !== 'undefined' ? new Headers() : {})
     this.ssrMode = config.ssrMode
     this.ssrPromises = []
     this.url = config.url
     this.fetch =
-      config.fetch || (typeof fetch !== 'undefined' && fetch && fetch.bind())
+      config.fetch ||
+      (typeof fetch !== 'undefined' && fetch && fetch.bind(this))
```

Comment:
no functional change in the way we call .bind?

Reply:
I checked and the default behaviour is to use `this` from the current scope so it should be the same. TypeScript just considers not passing a parameter as invalid

Reply:
thought as much, thanks for clarifying